Tutorial/WindowsVersion/tutorial.md

```diff
@@ -0,0 +1,39 @@
+1. Download SPTAG.
+    - git clone https://github.com/microsoft/SPTAG.git
+2. Build SPTAG. The build targets, IndexBuilder.exe, Server.exe, SPTAG.py, SPTAGClient.py, _SPTAG.pyd, and _SPTAGClient.pyd are all we need.
+3. Build the index
```

Comment:
how can user get the data used by this tutorial?

Reply:
Add a new step 3 to talk about that.